src/applications/coronavirus-vaccination/components/Form.jsx

```diff
@@ -111,11 +74,21 @@ function Form({ formState, updateFormData, router, isLoggedIn, profile }) {
       <h1 id="covid-vaccination-heading-form" className="no-outline">
         Fill out the form below
       </h1>
-      <p>
-        We’ll send you updates on how we’re providing COVID-19 vaccines across
-        the country—and when you can get your vaccine if you want one. You don't
-        need to sign up to get a vaccine.
-      </p>
+      {previouslySubmittedFormData ? (
+        <p>
+          Our records show you provided the information below on{' '}
+          {moment(previouslySubmittedFormData.createdAt).format('MMMM D, YYYY')}
+          . If you’d like to update your information, please make any updates
+          below and click <strong>Sign up to stay informed.</strong>
```

Comment:
@ncksllvn I think the button text changed based on Slack convo but I may not be fully up to date

Reply:
I _think_ this reflects latest changes. We did some UI updates on a call last night